src/components/Input/DropDown/DropDownInput.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import colors from "../../../theme/colors";
+
+const StyledDropDownItem = styled.span`
+  .dropdown__items & {
+    appearance: none;
+    height: 36px;
+    margin-left: 8px;
+    margin-right: 8px;
+    margin-top: 4px;
+    padding-left: 12px;
+    padding-right: 12px;
```

Comment:
Instead of having two separate padding attributes, you could combine them into `padding: 0 12px` for the same effect